csvqb/csvqb/models/cube/csvqb/components/attribute.py

```diff
@@ -3,13 +3,18 @@
 ----------
 """
 from dataclasses import dataclass, field
-from typing import Optional, List
+from typing import Optional, List, Union
 from abc import ABC, abstractmethod
 
 from csvqb.models.uriidentifiable import UriIdentifiable
+from sharedmodels.rdf.namespaces import QB
```

Comment:
Do we need the `QB` namespace in here? These models shouldn't really know anything about the RDF they serialise to.

Reply:
No, a route I went down and since backtracked.